README.md

```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
 
 **This package is very new and all of its features are experimental, not yet well tested, and subject to change without notice.**
 
+## The Zen of PreliZ
+* Being open source. 
+* Avoid automated solutions.
+* Separate tasks among humans and computers.
```

Comment:
Maybe we need to be more clear here, maybe something like this. or something more explicit

```suggestion
* Separate tasks between humans and computers, from each according to his ability.
```

Reply:
```suggestion
* Separate tasks between humans and computers, from each according to his abilities. So numerical tasks that are computationally demanding, error-prone or even tedious should be automatized as much as possible, while allowing the user to retain control of important decisions.
```

Reply:
```suggestion
* Separate tasks between humans and computers, so users can retain control of important decisions while numerically demanding, error-prone or tedious tasks are automatized.
```